delphes.sh

```diff
@@ -0,0 +1,51 @@
+package: Delphes
+version: "%(tag_basename)s"
+tag: master
+requires:
+  - ROOT
+  - HepMC
+  - pythia
+build_requires:
+  - CMake
+  - "GCC-Toolchain:(?!osx)"
+source: https://github.com/alisw/delphes.git
+#prepend_path:
+#  LD_LIBRARY_PATH: "$O2_ROOT/lib"
+#  DYLD_LIBRARY_PATH: "$O2_ROOT/lib"
+#  ROOT_INCLUDE_PATH: "$O2_ROOT/include"
+---
+#!/bin/bash -ex
+cmake $SOURCEDIR -DCMAKE_INSTALL_PREFIX=$INSTALLROOT       \
+                 ${CMAKE_GENERATOR:+-G "$CMAKE_GENERATOR"} \
+                 -DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE      \
+                 -DCMAKE_SKIP_RPATH=TRUE
+cmake --build . -- ${JOBS:+-j$JOBS} install
+
+#ModuleFile
+mkdir -p etc/modulefiles
+cat > etc/modulefiles/$PKGNAME <<EoF
+#%Module1.0
+proc ModulesHelp { } {
+  global version
+  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+}
+set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
+module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+# Dependencies
+module load BASE/1.0                                                                                \\
+            ${GCC_TOOLCHAIN_REVISION:+GCC-Toolchain/$GCC_TOOLCHAIN_VERSION-$GCC_TOOLCHAIN_REVISION} \\
+            ${PYTHIA_REVISION:+pythia/$PYTHIA_VERSION-$PYTHIA_REVISION}                             \\
+            ${HEPMC_REVISION:+HepMC/$HEPMC_VERSION-$HEPMC_REVISION}                                 \\
+            ${ROOT_REVISION:+ROOT/$ROOT_VERSION-$ROOT_REVISION}                                     \\
+
+# Delphes environment:
+set DELPHES_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
+setenv DELPHES_ROOT \$DELPHES_ROOT
```

Comment:
Do you really need this?

Reply:
yes, this is the way one will be able to point the location of the Delphes installation.
It will be needed to be able to find the Delphes "card" in case of running with provided cards.
It is necessary for developments that require Delphes classes.

Reply:
I guess since we will never pick this up from the system, it's fine...